docs/tutorials/Remark.adoc

```diff
@@ -0,0 +1,360 @@
+= AddressBook Level 3 - Tutorial
+:site-section: Tutorial
```

Comment:
`site-section` is used to determine which entry to highlight in the site navigation bar:

https://github.com/se-edu/addressbook-level35/blob/9aaee4dbfdb9f2ed8a421e556c652abe2afd3fca/docs/templates/_header.html.slim#L30-L41

So there's not much point in setting it to "Tutorial" if there is no corresponding entry there. You can: (1) Not set it at all, which is better than using an unrecognized value, (2) Add an entry on it to the site navigation bar (make sure to update the `site-section` docs!) or (3) Use an existing entry in the navigation bar

Reply:
I actually wrote some ruby to dump them into a `Tutorial` section. Think that got shafted when it broke something. Let me just remove it for now until I can remember what broke. iirc is_root? or function like that got removed in a later version of asciidoc...

Reply:
There was a period where I was struggling to get us up to asciidoctor 2.2 for asciidoctor-diagram to work properly but it ended up being a complete quagmire. I'll get around to it eventually(tm).